### ISSUE TITLE
REPORT FROM CUSTOMER:  Spaces in one-way box are not being removed before entering into SQL

### ISSUE THREAD

**Issue by JulieJager** [syncids/syncids-ui#449]
Spaces in one-way box are not being removed before entering into SQL which affects where or not the action happens.  

Customer reported references not flowing from one-way instructions.  After investigating, I found literal spaces between the matter names and removed them.  After removing them, the references flowed properly.  Could we stripe out any spaces added in this box before entering into SQL?  Thanks!

**Comment by JulieJager** [comment]
More info from customer:

When I was just syncing other matters, the system automatically entered a space between matters when I picked the new one from the drop down list that appeared.

I was one way importing new matter RAP626.GBP into RAP626.Con.  After typing RAP626, ‘RAP626.GBP’ appeared as a choice.  I clicked on it and when the system added it, it added a space between RAP626.PCT and RAP626.GBP, even though I started typing directly after RAP626.PCT.  I then had to go back in and manually remove the added space.  Don’t know if the same happens if I type the entire matter myself without choosing it from the drop down choices.
